packages/ui/src/components/accordion/accordion.vue

```diff
@@ -90,29 +88,25 @@
 
   const classes = computed(() => {
     return {
-      'acv-accordion': true,
```

Comment:
@leonid can we consider moving all classes to template? I think we should create custom eslint rule to check that topmost class is prefixed, and it will be much easier if all classess are inside the template

Reply:
Yes, we can do it. You can can create a task and pull request. 
This eslint rule should have fixes section.